src/configs/sepolia.rs

```diff
@@ -20,8 +20,8 @@ pub const BELLATRIX_FORK_VERSION: Version = [144, 0, 0, 113];
 pub const BELLATRIX_FORK_EPOCH: Epoch = 100;
 pub const CAPELLA_FORK_VERSION: Version = [3, 0, 16, 32];
 pub const CAPELLA_FORK_EPOCH: Epoch = FAR_FUTURE_EPOCH;
-pub const EIP4844_FORK_VERSION: Version = [4, 0, 16, 32];
-pub const EIP4844_FORK_EPOCH: Epoch = FAR_FUTURE_EPOCH;
+pub const DENEB_FORK_VERSION: Version = [4, 0, 16, 32];
+pub const DENEB_FORK_EPOCH: Epoch = FAR_FUTURE_EPOCH;
```

Comment:
the `fork_version` looks incorrect (seems like the goerli values)

ill go fix in another PR and also set the sepolia fork epoch, since we know it now

Reply:
I've created a PR to update the sepolia values for Capella
https://github.com/ralexstokes/ethereum-consensus/pull/194

Doesn't look like the Deneb config are out yet, so I'll leave them as it is.